### PR TITLE
Fixed Tags and changepassword

### PR DESCRIPTION
**Pull request by GlennPegdenFLT**
Fixed addition of DNS Tags (typo resulted in syntax error)

Removed createsuperuser, not only was it superfluous, so it always fails, but that failure means changepassword doesn't run, so the admin can never log in

### CODE REVIEW

**Comment by gsilvapt** [comment]
Thanks for your contribution, @GlennPegdenFLT. Can you confirm which version of Django you installed in your environment with `pip list | grep Django`? I'm asking because this used to work and _maybe_ the API changed in Django 4. Just want to validate that's the case.

**Comment by GlennPegdenFLT** [comment]
I actually just came here to cancel the PR as I realised the changepassword part may be wrong (in fact, probably is wrong) as I was unintentionally using a mysql volume from an older install of surface (I'd assumed it was a clean install).

As you've spotted though, the tags part should still be valid though. 

Sorry about that.

File: dev/box_setup.sh [hunk 1]
```diff
@@ -12,7 +12,9 @@ function dexec() {
 dexec surface ./manage.py migrate
 
 # create local admin (ignore failure as user might already exist)
-dexec surface ./manage.py createsuperuser --noinput --username admin --email admin@localhost && dexec surface ./manage.py changepassword admin
+
+#dexec surface ./manage.py createsuperuser --noinput --username admin --email admin@localhost
+dexec surface ./manage.py changepassword admin
```
Comment (gsilvapt) [inline dev/box_setup.sh:17]:
```suggestion

dexec surface ./manage.py createsuperuser --noinput --username admin --email admin@localhost
```

As discussed then, reverting this change to the original docker execution.

**Comment by gsilvapt** [comment]
I am unsure whether this permission error is because this is coming from a fork (and therefore you cannot push to the project's registry to avoid polluting with malicious code), or simply because of outdated actions. We will get back to this an upgrade all actions to rule out the issues - if needed, I'll reopen this change on my own PR - thank you very much.

**Comment by gsilvapt** [comment]
Superseded by #172

**Pull request closed without merging**